src/v/rpc/include/rpc/errc.h

```diff
@@ -47,6 +47,10 @@ struct errc_category final : public std::error_category {
             return "rpc::errc::missing_node_rpc_client";
         case errc::client_request_timeout:
             return "rpc::errc::client_request_timeout";
+        case errc::service_error:
+            return "rpc::errc::service_error";
+        case errc::method_not_found:
+            return "rpc::errc::method_not_found";
```

Comment:
nice cleanup. could/should this switch/case be exhaustive (i.e. no `default`)?

Reply:
Yeah it's probably best to do that. I've fixed this now.